hack/tools/requirements.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o nounset
+
+HELM_VERSION=v3.8.2
+ISTIO_VERSION=1.8.0
+JSON_SCHEMA_GENERATOR_VERSION=0.2.1
+KUBE_VERSION=1.22.0
+KUSTOMIZE_VERSION=3.2.0
```

Comment:
do we use kustomize ?
and others ?

Reply:
I copied the file as is from fybrik. I'll remove references for unneeded tools.